v22.2/manage-certs-vault.md

```diff
@@ -570,14 +570,14 @@ Download the CockroachDB executable from `binaries.cockroachdb.com`.
 {% include_cached copy-clipboard.html %}
 ```shell
 function install_roach_on_node {
-    gcloud compute ssh $1 --command 'if [[ ! -e cockroach-v21.2.4.linux-amd64 ]];
+    gcloud compute ssh $1 --command 'if [[ ! -e cockroach-{{ page.release_info.version }}.linux-amd64 ]];
```

Comment:
Guess: No longer need specific previous minor if said minor is two majors in the past.